exonum-java-binding/package_app.sh

```diff
@@ -6,6 +6,22 @@
 # Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
+function build-ejb-app-macos() {
+    # Set RUSTFLAGS to adjust RUNPATH of the binary.
+    export RUSTFLAGS="-C link-arg=-Wl,-rpath,@executable_path/lib/native"
+    echo "RUSTFLAGS=${RUSTFLAGS}"
+    mvn package --activate-profiles app-packaging -pl :exonum-java-binding-core -am -DskipTests \
```

Comment:
Why do we skip tests when building an application (i.e., why do we allow to build a package that does not pass even unit/ITs)? Shan't there be a TODO to fix that?